python/unicornafl/__init__.py

```diff
@@ -228,9 +192,25 @@ def uc_afl_fuzz_custom(uc: Uc,
     cb3 = ctypes.cast(UC_AFL_FUZZ_CALLBACK_CB(
         _fuzz_callback_cb), UC_AFL_FUZZ_CALLBACK_CB)
 
-    err = _uc2afl.uc_afl_fuzz_custom(uc._uch, input_file.encode("utf-8"), cb1, cb3, 
-        cb2, always_validate, persistent_iters, ctypes.cast(idx, ctypes.c_void_p))
-
+    if isinstance(input_file, str):
+        input_file = ctypes.create_string_buffer(input_file.encode('utf-8'))
```

Comment:
Shouldn't this be:

```suggestion
        input_file = \
            ctypes.cast(
                ctypes.create_string_buffer(input_file.encode('utf-8')),
                ctypes.c_void_p
            ).value
```

See same for case below.

Reply:
Yeah, the changes seem not committed =(.

Reply:
I'd argue that a testcase for the `uc_afl_fuzz_custom` route would be beneficial as well - are there any uncommited changes on the rust part of `uc_afl_fuzz_custom`? Otherwise I will debug this route a little bit and see if I find anything further

Reply:
> I'd argue that a testcase for the `uc_afl_fuzz_custom` route would be beneficial as well - are there any uncommited changes on the rust part of `uc_afl_fuzz_custom`? 

Contributions are highly welcome =).

Reply:
Done for the `input_file` handling.

Reply:
Marvellous :)